tests/h/services/list_organizations_test.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+
+from h.services.list_organizations import ListOrganizationsService
+from h.services.list_organizations import list_organizations_factory
+from h.services.organization import organization_factory
+
+
+class TestListOrganizations(object):
+
+    def test_returns_organizations_from_all_authorities_if_no_authority_specified(
+            self,
+            svc,
+            organizations,
+            alternate_organizations):
+
+        results = svc.organizations()
+
+        assert '__default__' in [org.pubid for org in results]
+        assert not (set(organizations + alternate_organizations) - set(results))
```

Comment:
I would recommend trying to write asserts in the form `assert actual == expected` where possible as this gives more helpful error messages when the test fails, compared to more complex expressions.

In this case you could replace these two asserts with something like:

```py
expected = organizations + alternate_organizations + [Organization.default(db_session)]
assert set(expected) == set(results)
```